app/preprints/-components/preprint-institutions/institution-select-list/component.ts

```diff
@@ -1,16 +1,28 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import Intl from 'ember-intl/services/intl';
+import InstitutionsManagerComponent from '../institution-manager/component';
 
-import { layout } from 'ember-osf-web/decorators/component';
-import { InstitutionsManager } from 'osf-components/components/editable-field/institutions-manager/component';
-import styles from './styles';
-import template from './template';
 
-@layout(template, styles)
-export default class InstitutionSelectList extends Component {
+/**
+ * The Institution Select List Args
+ */
+interface InstitutionSelectListArgs {
+    manager: InstitutionsManagerComponent;
+}
+
+export default class InstitutionSelectList extends Component<InstitutionSelectListArgs> {
+    @service intl!: Intl;
+
     // Required
-    manager!: InstitutionsManager;
+    manager = this.args.manager;
+
+    public get displayComponent(): boolean {
+        return this.args.manager.institutions.length > 0;
```

Comment:
Should this component be visible if the preprint has any affiliated institutions as well? It looks like this condition just checks if the user has any affiliated institutions in this case.

Reply:
Ok, so upon further reading, it looks like this edit flow should only show the institution's that the user is affiliated with, so this logic is right.